### ISSUE TITLE
Can't output a string with a semicolon in it with #echo. Is this a bug?

### ISSUE THREAD

**Issue by dzpao** [scandum/tintin#139]
I've tried various methods and can't output a string with a semicolon in it with `#echo`. So is this a bug?

All commands except `#echo` accept semicolons as part of the string normally.

```
zsh ❯ ./tt-master -G -e "#echo {FOO: %s} {hello;world}; #end {\}"
FOO: hello
zsh ❮ ./tt-master -G -e '#var foo {hello;world}; #echo {FOO: %s} {$foo}; #end {\}'
FOO: hello
zsh ❯ ./tt-master -G -e '#var foo {hello;world}; #replace foo {;} {:}; #echo {FOO: %s} {$foo}; #end {\}'
FOO: hello:world
```

**Comment by scandum** [comment]
That's a bit odd indeed, added it to my todo list.

**Comment by scandum** [comment]
Should be fixed in 2.02.30.
